docs/guides/other-guides/monitor-ngrok/monitor-ngrok.md

```diff
@@ -0,0 +1,312 @@
+- Stop the ngrok Docker container in the terminal with <kbd>Ctrl</kbd><kbd>+</kbd><kbd>C</kbd>.
+- Create a file named `policy.yml` with the code below.
+
+  ```yml
+  on_http_request:
+    - actions:
+      - type: rate-limit
+        config:
+          name: 1 request per minute
+          algorithm: sliding_window
+          capacity: 1
+          rate: 1m
+          bucket_key:
+            - "hasReqHeader('host') ? getReqHeader('host')[0] : 'unknown'"
+  ```
+
+- Run the agent again with the command below. Replace the authentication token and the URL with your token and URL (using `https`, not `http`).
+
+  ```sh
+  docker run -it --rm --platform=linux/amd64 --network=ngrokTest -v ".:/app" -w "/app" -e NGROK_AUTHTOKEN=Y0urS3cr3t ngrok/ngrok:3.22.0-alpine-amd64 http http://api --traffic-policy-file policy.yml --url https://massive-pelican-arguably.ngrok-free.app
+  ```
+
+- Browse to the API URL and refresh the page a few times. Notice that the page stops responding.
+- On the ngrok Traffic Inspector page, note that the error code **`429`** (rate limit) was returned and that the duration of the request was instant and caused no load on your API.
+
+  ![Rate limit](./img/rateLimit.png)
+
+- Edit the `policy.yml` file and change `capacity` to `10`, then restart the ngrok container.
+- In the Traffic Inspector, click on one of the **`429`** events, then click **Replay**. Note that the request now responds without error because the rate limit has been increased.
+
+  ![Replay after rate limit adjustment](./img/replayLimit.png)
+
+## Monitor events
+
+In this section, you'll learn how to export ngrok events to the Datadog monitoring service.
+
+There are two [types of events](https://ngrok.com/docs/obs/events/reference): standard traffic events (requests to your API) and audit events (changes to secret keys and URLs). For this simple example, you'll monitor a traffic event.
+
+Before adding an event subscription, you need somewhere to send events:
+
+- Sign up for a [Datadog](https://www.datadoghq.com) trial account. (You will eventually need a paid Datadog plan to continue using logs and monitors.)
+
+  You can enter anonymous dummy values instead of your personal information for all the Datadog registration fields except your email address, which you need to confirm. You also can't skip the third step of the sign-up process, in which you create a Datadog agent somewhere.
+
+- In Step 3 of the Datadog sign-up, click on **Docker** in the sidebar. Copy and paste the given command into a terminal and run it.
+
+  ![Join Datadog](./img/joinDatadog.png)
+
+- Once the Datadog site notices your agent is running, click **Finish** at the bottom of the page.
+- You now no longer need the agent running locally and can remove the Datadog container with the command below.
+
+  ```sh
+  docker stop dd-agent
+  docker remove dd-agent
+  docker image rm gcr.io/datadoghq/agent:7
+  ```
```

Comment:
Can you get an API key this way? https://docs.datadoghq.com/account_management/api-app-keys/#add-an-api-key-or-client-token